### DISCUSSION TITLE
Get avatar URL

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6791] [Ideas]
Currently we only support `get_avatar()` in `Pods::field()` or `{@avatar}` magic tags.
We should also support `avatar_url` or `avatar_src` as a field input.

**Comment by jimtrue** [comment]
@JoryHogeveen Shouldn't these just use the `_img` or `_src` helpers?

**Comment by JoryHogeveen** [comment]
@jimtrue thats even better :)
